app/routes/__init__.py

```diff
@@ -1,19 +1,24 @@
-from flask import Flask, Blueprint
+from flask import Blueprint, Flask
+
+bp_api = Blueprint("bp_api", __name__, url_prefix="/api")
 
-bp_api = Blueprint("bp_api", __name__,url_prefix='/api')
 
 def init_app(app: Flask):
 
-    from .address_blueprint import bp_address 
+    from .address_blueprint import bp_address
```

Comment:
Maybe re-organize those imports inside the init app to above all the logic, when we use black, it add a white space after each import, in that way the init app would be too big.

Reply:
Done.